<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyAddDialog</class>
 <widget class="QWidget" name="KeyAddDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Recipient</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="skCerts">
      <attribute name="title">
       <string>SK Certificates</string>
      </attribute>
      <layout class="QVBoxLayout" name="skCertsLayout">
       <item>
        <layout class="QHBoxLayout" name="searchLayout">
         <item>
          <widget class="QComboBox" name="searchType">
           <item>
            <property name="text">
             <string>Personal code:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Company name:</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="searchContent"/>
         </item>
         <item>
          <widget class="QPushButton" name="search">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="skView">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Owner</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Issuer</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Expiry date</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="skCertsAddLayout">
         <item>
          <widget class="QProgressBar" name="progress">
           <property name="maximum">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="skCertsAddSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="keyAddStatus"/>
         </item>
         <item>
          <widget class="QPushButton" name="add">
           <property name="text">
            <string>Add cert</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="used">
      <attribute name="title">
       <string>Used certificates</string>
      </attribute>
      <layout class="QVBoxLayout" name="storeLayout">
       <item>
        <widget class="QTreeWidget" name="usedView">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Owner</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Issuer</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Expiry date</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="removeLayout">
         <item>
          <spacer name="removeSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="remove">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>searchContent</sender>
   <signal>returnPressed()</signal>
   <receiver>search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KeyAddDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
